### PR TITLE
Added Hyperlink to reference

### PR DESCRIPTION
**Pull request by rrrrrrmb**
#### References to other Issues or PRs

None

#### Brief description of what is fixed or changed
The link given for downloading the reference given in [category module](https://github.com/sympy/sympy/blob/master/doc/src/modules/categories.rst) was not a hyperlink.I have changed it to a hyperlink for better usage.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: doc/src/modules/categories.rst [hunk 1]
```diff
@@ -11,13 +11,12 @@ whether they are commutative or not.
 
 The general reference work this module tries to follow is
 
-  [JoyOfCats] J. Adamek, H. Herrlich. G. E. Strecker: Abstract and
-              Concrete Categories. The Joy of Cats.
+  [JoyOfCats] J. Adamek, H. Herrlich. G. E. Strecker: Abstract and Concrete Categories. The Joy of Cats.
```
Comment (eric-wieser) [inline doc/src/modules/categories.rst:14]:
```suggestion
.. [JoyOfCats] J. Adamek, H. Herrlich. G. E. Strecker: Abstract and Concrete Categories. The Joy of Cats.
```

Comment (mdb571) [reply doc/src/modules/categories.rst:14]: Okay on it

**Comment by eric-wieser** [comment]
Thanks!

**Comment by moorepants** [comment]
It looks like master failed to build on CI after this merge.

**Comment by eric-wieser** [comment]
Looks to me like a network failure unrelated to the patch.

**Pull request merged**